guides/source/action_view_overview.md

```diff
@@ -1505,6 +1505,266 @@ strip_tags("<b>Bold</b> no more!  <a href='more.html'>See more</a>")
 
 NB: The output may still contain unescaped '<', '>', '&' characters and confuse browsers.
 
+### UrlHelper
+Provides a set of methods for making links and getting URLs that
+depend on the routing subsystem (see ActionDispatch::Routing).
+This allows you to use the same format for links in views
+and controllers.
```

Comment:
I think `Provides methods for making links and getting URLs that
depend on Rails' routing subsystem.` is concise enough.